packages/muelu/doc/TutorialReadTheDocs/pages/level_smoothers.rst

```diff
@@ -0,0 +1,148 @@
+
+    - .. figure:: pics/2level_10jac09.png
+
+        2 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/2level_100jac09.png
+
+        2 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/3level_1jac09.png
+
+        3 levels with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/3level_10jac09.png
+
+        3 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/3level_100jac09.png
+
+        3 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+.. _level_smoothers/figure_5vcycles:
+
+.. list-table:: 2D Laplace equation on 50 x 50 mesh after 5 V-cycle with an AMG multigrid solver and Jacobi smoothers on all multigrid levels. (2 processors) 
+
+  * - .. figure:: pics/5sweeps_1level_1jac09.png
+
+        1 level with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_1level_10jac09.png
+
+        1 level with 10 Jacobi sweeps (:math:`\omega=0.9`) 
+
+    - .. figure:: pics/5sweeps_1level_100jac09.png
+
+        1 level with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_2level_1jac09.png
+
+        2 levels with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_2level_10jac09.png
+
+        2 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_2level_100jac09.png
+
+        2 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_3level_1jac09.png
+
+        3 levels with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_3level_10jac09.png
+
+        3 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_3level_100jac09.png
+
+        3 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+
+.. admonition:: Exercise 1
+
+    Play around with the smoother parameters and study their effect on the error plot and the convergence of the preconditioned cg method. For all available smoothing options and parameters refer to the MueLu user guide ([1]_). Hint: use **unsmoothed** transfer operator basis functions (i.e., **multigrid algorithm = unsmoothed**) to highlight the effect of the level smoothers.
+
+.. admonition:: Exercise 2
+
+    Use the following parameters to solve the :math:`50\times 50` Laplace 2D problem on 2 processors 
+
+    .. literalinclude:: ../../../test/tutorial/s1_easy_exercise.xml
+        :language: xml
+  
+  That is, we change to smoothed aggregation AMG. You can find the xml file also in **../../../test/tutorial/s1_easy_exercise.xml**. Run the example on 2 processors and check the number of linear iterations and the solver timings in the screen output. Can you find smoother parameters which reduce the number of iterations? Can you find smoother parameters which reduce the iteration timings?
+
+
+Footnotes
+=========
+.. [1] A. Prokopenko, J.J. Hu, T.A. Wiesner, C.M. Siefert and R.S. Tuminaro **MueLu User’sGuide 1.0 (Trilinos Version 11.12)**, SAND2014-18874, 2014
```

Comment:
Update the reference to the most recent user guide.

Reply:
Done.